augly/video/functional.py

```diff
@@ -2137,3 +2139,58 @@ def vstack(
         helpers.get_metadata(metadata=metadata, function_name="vstack", **func_kwargs)
 
     return output_path or video_path
+
+
+def augment_audio(
```

Comment:
nit: we usually keep these in alphabetical order, so please move this up to between `audio_swap` & `blend_videos` :)